From 8a5987740e56f7ac9c01519f023cbddb77cd3ce9 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 24 Aug 2018 12:47:06 +0300
Subject: [PATCH] BTL [uart]: add nops before each write to UART

Need this to avoid override of characters since BTL
machine is to fast for our uart
---

diff --git a/board/habanalabs/common/btl/uart.s b/board/habanalabs/common/btl/uart.s
index 1a69b79..49eeda2 100644
--- a/board/habanalabs/common/btl/uart.s
+++ b/board/habanalabs/common/btl/uart.s
@@ -27,16 +27,9 @@
 
 /* log BTLID\n\r */
 cfg mmPSOC_UART_0_DATA, 0X0A
-nop
-nop
-nop
-nop
-nop
-nop
-nop
-nop
-nop
-nop
+#include "nop.s"
 cfg mmPSOC_UART_0_DATA, BTLID
+#include "nop.s"
 cfg mmPSOC_UART_0_DATA, 0X0A
+#include "nop.s"
 cfg mmPSOC_UART_0_DATA, 0X0D
